x/auth/client/cli/genaccounts.go

```diff
@@ -33,6 +36,7 @@ func AddGenesisAccountCmd(ctx *server.Context, cdc *codec.Codec,
 			config := ctx.Config
```

Comment:
Hmmm, so it might make sense to remove this command altogether. The ADR states the application will define and mount this command(s).